pimd/pim_nht.c

```diff
@@ -430,6 +430,9 @@ int pim_ecmp_nexthop_search(struct pim_instance *pim,
 	if (!pnc || !pnc->nexthop_num || !nexthop)
 		return 0;
 
+	memset(&nbrs, 0, sizeof(nbrs));
```

Comment:
shouldn't this be memset( &nbrs[0]... and &ifps[0]?

Reply:
Oh, thank you, I'll check it, sorry! :-)

Reply:
@donaldsharp It is equivalent, because the address of the first element is the same as the address of the whole data structure.

Reply:
Correct, this is fine.